astropy/cosmology/_src/traits/curvature.py

```diff
@@ -0,0 +1,76 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Critical density.
+
+This is private API. See `~astropy.cosmology.traits` for public API.
+
+"""
+
+__all__ = ["CurvatureComponent"]
+
+__doctest_requires__ = {"CurvatureComponent.Ok0": ["numpy>=2.0"]}
+
+from typing import Any
+
+import numpy as np
+import numpy.typing as npt
+from numpy.typing import ArrayLike
+
+from astropy.cosmology._src.utils import aszarr, deprecated_keywords
+from astropy.units import Quantity
+
+
+class CurvatureComponent:
+    """The object has attributes and methods related to the global curvature."""
+
+    Ok0: float | np.floating[Any]
+    """Omega curvature; the effective curvature density/critical density at z=0."""
+
+    @deprecated_keywords("z", since="7.0")
+    def Ok(self, z: Quantity | ArrayLike) -> npt.NDArray | np.floating[Any]:
+        """Return the equivalent density parameter for curvature at redshift ``z``.
+
+        Parameters
+        ----------
+        z : Quantity-like ['redshift'], array-like
+            Input redshift.
+
+            .. versionchanged:: 7.0
+                Passing z as a keyword argument is deprecated.
+
+        Returns
+        -------
+        Ok : ndarray
+            The equivalent density parameter for curvature at each redshift.
+
+            .. versionchanged:: 7.1
+                Always returns a numpy object, never a `float`.
```

Comment:
Why is wasn't this added in 7.1 documentation?

Reply:
Because it should be 7.2 lol. Forgot we already released 7.1